app/stats/Metrics.tsx

```diff
@@ -96,13 +98,38 @@ const MetricBox = ({
     data: samples.map((s) => s.label),
     scaleType: "band",
   };
+
+  const calculateTrend = (data: Array<number>) => {
+    const lastValue: number = data[data.length - 1];
+    const penultimateValue: number = data[data.length - 2];
+    const trend = ((lastValue - penultimateValue) / penultimateValue) * 100;
+    const twoDecimalsTrend = Math.round(trend * 100) / 100;
+    return twoDecimalsTrend;
+  };
+
+  const trend = calculateTrend(data);
+
+  const shouldBeGreen = (metric: MetricDefinition, trend: number) => {
+    return (
+      (trend >= 0 && metric.positiveDirection === "moreIsBetter") ||
+      (trend <= 0 && metric.positiveDirection === "lessIsBetter")
+    );
+  };
+
+  const isGreen: boolean = shouldBeGreen(metric, trend);
+
   return (
     <Link
       key={metric.slug}
       className="flex flex-col overflow-clip rounded-lg border-2 border-gray bg-pebble hover:border-it hover:shadow-lg"
       href={Route.toMetric(metric)}
     >
       <div className="p-4">
+        <h4
+          className={`typo-caption ${isGreen ? "text-green-600" : "text-red-600"}`}
```

Comment:
U těch podmíněných stylů mrkni na knihovnu [clsx](https://www.npmjs.com/package/clsx), kterou používáme. Trochu to zjednoduší, není tam pak tolik složité interpolace.